kotlin-react-router/karakum/plugins/convertReactQualifiedName.js

```diff
@@ -0,0 +1,21 @@
+const ts = require("typescript");
```

Comment:
Why it can't be `import`?

Reply:
Because right now I compile my source code of Karakum converter as CommonJS, I will try to migrate to Kotlin or TS for plugins in future.